doc/getting_started/getting_started.rst

```diff
@@ -6,6 +6,17 @@ Getting Started Guide
 Use this guide to get started with your :ref:`Zephyr <introducing_zephyr>`
 development.
 
+.. note::
```

Comment:
You're right.  I'm indeed not fond of this command prefix notation, particularly when the commands are interspersed as shown in some example here and can lead to confusion by the reader.  I'd much rather see separate "for Windows" and "for Linux/macOS" code blocks than what's being proposed.  (And not include this note.)